api/zyre.api

```diff
@@ -284,6 +284,12 @@
         Return socket for talking to the Zyre node, for polling
         <return type = "zsock" />
     </method>
+	
+	<method name = "socket_zmq">
+        Return underlying ZMQ socket for talking to the Zyre node, 
+        for polling with libzmq (base ZMQ library)
+        <return type = "void *" />
```

Comment:
Can you try to change the type to "anything" or perhaps "sockish"? See: https://github.com/zeromq/zproject#api-types